app/application.py

```diff
@@ -363,15 +373,6 @@ def register_enter_your_details():
             # TODO A redirect to a page that helps the user
 
         except requests.exceptions.Timeout:
-            print("Timeout error. Is the OAuth Server overloaded?")
-            # TODO A redirect to a page that helps the user
-        except requests.exceptions.RequestException as e:
-            # TODO catastrophic error. bail. A page that tells the user something horrid has happeded and who to inform
-            print(e)
-            app.logger.debug("There seems to be no server listening on this connection?")
-            # TODO A redirect to a page that helps the user
-
-        except requests.exceptions.Timeout:
             app.logger.debug("Timeout error. Is the OAuth Server overloaded?")
             # TODO A redirect to a page that helps the user
         except requests.exceptions.RequestException as e:
```

Comment:
After the except is caught we don't do anything. Not even a 'print' ! Can you put at least a #TODO or/and logger.warning/logger.error message. :-) Thank you.